src/common/MDSCapMatch.h

```diff
@@ -0,0 +1,169 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*- 
```

Comment:
This file can be in `src/mds`. The client reads headers from `src/mds/` already.

Reply:
Sure, will fix it.